README.md

```diff
@@ -1,5 +1,7 @@
 # Django Query Inspector
 
+TEST
```

Comment:
is this really necessary?

Reply:
Absolutely necessary.

Reply:
You make a reasoned argument.  I'll consider it.